backend/src/main/java/com/alibaba/higress/console/controller/dto/TlsCertificate.java

```diff
@@ -44,4 +48,22 @@ public class TlsCertificate {
 
     @JsonFormat(pattern = "yyyy/MM/dd HH:mm:ss")
     private LocalDateTime validityEnd;
+
+    public boolean valid() {
+        if (StringUtils.isAnyBlank(name, version, cert, key)) {
+            return false;
+        }
+        if (CollectionUtils.isEmpty(domains)) {
+            return false;
+        }
+        for (String domain : domains) {
+            if (!ValidateUtil.checkDomain(domain)) {
+                return false;
+            }
+        }
+        if (Objects.isNull(validityStart) || Objects.isNull(validityEnd)) {
+            return false;
+        }
+        return validityEnd.isAfter(validityStart);
```

Comment:
domains, validityStart and validityEnd are extracted from cert data. Client doesn't submit them.